.github/workflows/main.yml

```
@@ -0,0 +1,29 @@
+name: Docs Publish
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        language: [de_DE, ja_JP]
```

Can we leave an `if` condition to check for `skip ci` in the commit message? (Reference: https://github.com/marketplace/actions/skip-based-on-commit-message)

```suggestion
        language: [de_DE, ja_JP]
    if: "!contains(github.event.head_commit.message, 'skip ci')"
```